import React, {Component} from 'react';
import styles from './references.module.scss';
import overallStyles from '../overall.module.scss';
import Collapsible from '../../utils/Collapsible';

class References extends Component {
    render() {
        return (
            <div className={overallStyles.container}>
                <div className={overallStyles.text_div} style={{marginBottom: 40}}>
                    <span className={overallStyles.text_heading} style={{paddingBottom: 0}}>References</span>
                </div>
                {/* <div>
                    <div className={styles.illustration}></div>
                </div> */}
                <Collapsible open title="Microfluidic Chip Design">
                    <div className={overallStyles.description}>
                        <p>
                        <ul className={overallStyles.description} style={{marginTop: 0}}>
                            <li><p>Aghvamia, S. A., Opathalagea, A., Zhang, Z. K., Ludwig, M., Heymann M., Norton, M., Wilkins, N., &amp; Fraden, S. (August 2017). Rapid prototyping of cyclic olefin copolymer (COC) microfluidic devices. <em>Sensors and Actuators B: Chemical, 247</em>(4), <a href="https://doi.org/10.1016/j.snb.2017.03.023">https://doi.org/10.1016/j.snb.2017.03.023</a>.</p></li>
                            <li><p>Barley, J. (n.d.). <em>Basic principles of freeze drying</em>. SP Scientific. Retrieved October 10, 2021, from <a href="https://www.spscientific.com/freeze-drying-lyophilization-basics/">https://www.spscientific.com/freeze-drying-lyophilization-basics/</a></p></li>
                            <li><p>Beyzavi, K., Hampton, S., Kwasowski, P., Fickling, S., Marks, V., &amp; Clift, R. (1987). Comparison of horseradish peroxidase and alkaline phosphatase-labelled antibodies in enzyme immunoassays. <em>Annals of Clinical Biochemistry</em>, <em>24</em>, 145-152. doi:10.1177/000456328702400204</p></li>
                            <li><p>Chang, Y., Xu, J., &amp; Zhang, Q. (2017). Microplate magnetic chemiluminescence immunoassay for detecting urinary survivin in bladder cancer. <em>Oncology Letters</em>, <em>14</em>(4), 4043-4052. doi:10.3892/ol.2017.6675</p></li>
                            <li><p>Chattopadhyay, K., &amp; Mazumdar, S. (2000). Structural and conformational stability of horseradish peroxidase:  effect of temperature and pH. <em>Biochemistry</em>, <em>39</em>(1), 263-270. doi:10.1021/bi990729o</p></li>
                            <li><p>Eamudomkarn, C., Sithithaworn, P., Kamamia, C., Yakovleva, A., Sithithaworn, J., Kaewkes, S., Techasen, A., Loilome, W., Yongvanit, P., Wangboon, C., Saichua, P., Itoh, M., &amp; Bethony, J. M. (2018). Diagnostic performance of urinary IGG Antibody Detection: A novel approach for population screening of strongyloidiasis. <em>PloS One</em>, <em>13</em>(7), e0192598-e0192598. doi:10.1371/journal.pone.0192598 <a href="https://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0192598">https://journals.plos.org/plosone/article?id=10.1371%2Fjournal.pone.0192598</a>.</p></li>
                            <li><p>Ghosh, S., &amp; Ahn, C. (2019). Lyophilization of chemiluminescent substrate reagents for high-sensitive microchannel-based lateral flow assay (MLFA) in point-of-care (POC) diagnostic system. <em>Analyst</em>, <em>144</em>(6): 2109.</p></li>
                            <li><p>Ghosh, S., Aggarwal, K., Vinitha, T. U., Nguyen, T., Han, J., &amp; Ahn, C. H. (2020). A new microchannel capillary flow assay (MCFA) platform with lyophilized chemiluminescence reagents for a smartphone-based POCT detecting malaria. <em>Microsystems &amp; Nanoengineering</em>, <em>6</em>(1), 5. doi:10.1038/s41378-019-0108-8</p></li>
                            <li><p>Heo, Y., Shin, K., Park, M. C., &amp; Kang, J. Y. (2021). Photooxidation-induced fluorescence amplification system for an ultra-sensitive enzyme-linked immunosorbent assay (ELISA). <em>Scientific Reports</em>, <em>11</em>(1), 5831. doi:10.1038/s41598-021-85107-7</p></li>
                            <li><p>Kim, D., &amp; Herr, A. E. (2013). Protein immobilization techniques for microfluidic assays. <em>Biomicrofluidics</em>, <em>7</em>(4), 041501. <a href="https://doi.org/10.1063/1.4816934">https://doi.org/10.1063/1.4816934</a></p></li>
                            <li><p>Liu, Z., &amp; Chen, X. (2016). Simple bioconjugate chemistry serves great clinical advances: Albumin as a versatile platform for diagnosis and precision therapy. <em>Chemical Society Reviews, 45</em>(5), 1432-1456. doi:10.1039/c5cs00158g</p></li>
                            <li><p>Mero, A., Clementi, C., Veronese, F. M., &amp; Pasut, G. (2011). Covalent Conjugation of Poly(Ethylene Glycol) to Proteins and Peptides: Strategies and Methods. <em>Methods in Molecular Biology Bioconjugation Protocols</em>, 95-129. doi:10.1007/978-1-61779-151-2_8</p></li>
                            <li><p>Oh, H., Cerchiari, A., Sørensen, D., Mon, T., &amp; Smith, C. (2013). Rolling Circle Amplification on Biotin-Streptavidin Complexes Immobilized to Activated Cyclic Polyolefin Surfaces. <em>Materials Sciences and Applications, 4</em>(9), pp. 538-548. doi: 10.4236/msa.2013.49066.</p></li>
                            <li><p>Olanrewaju, A., Beaugrandm M., Yafia, M., Juncker, D. Capillary microfluidics in microchannels: from microfluidic networks to capillaries circuits. Lab on a Chip 16, 2323-2347 (2018). <a href="https://doi.org/10.1039/C8LC00458G">https://doi.org/10.1039/C8LC00458G</a></p></li>
                            <li><p>Osman, A. M., Zomer, G., Laane, C., &amp; Hilhorst, R. (2000). Comparative studies of the chemiluminescent horseradish peroxidase-catalysed peroxidation of acridan (GZ-11) and luminol reactions: Effect of pH and scavengers of reactive oxygen species on the light intensity of these systems. <em>Luminescence : The Journal of Biological and Chemical Luminescence</em>, <em>15</em>(3), 189-197. doi:10.1002/1522-7243(200005/06)15:3{'<'}189::AID-BIO585{'>'}3.0.CO;2-A </p></li>
                            <li><p>Pal, K., Paulson, A. T., &amp; Rousseau, D. (2013). Biopolymers in CONTROLLED-RELEASE delivery systems. <em>Handbook of Biopolymers and Biodegradable Plastics</em>, 329–363. <a href="https://doi.org/10.1016/b978-1-4557-2834-3.00014-8">https://doi.org/10.1016/b978-1-4557-2834-3.00014-8</a></p></li>
                            <li><p>Roda, A. (2011). Chemiluminescence and bioluminescence past, present and future. <em>Royal Society of Chemistry</em>.</p></li>
                            <li><p>Roughton, B. C., Iyer, L. K., Bertelsen, E., Topp, E. M., &amp; Camarda, K. V. (2013). Protein aggregation and lyophilization: Protein structural descriptors as predictors of aggregation propensity. <em>Computers &amp; chemical engineering, 58</em>: 369–377. Retrieved October 10, 2021, from <a href="https://doi.org/10.1016/j.compchemeng.2013.07.008">https://doi.org/10.1016/j.compchemeng.2013.07.008</a></p></li>
                            <li><p>Sticker, D., Geczy, R., Häfeli, U. O., &amp; Kutter, J.P. (2020 March 4). Thiol-Ene Based Polymers as Versatile Materials for Microfluidic Devices for Life Sciences Applications. <em>ACS Applied Material Interfaces</em>; 12*(9):10080-10095. doi: 10.1021/acsami.9b22050. Epub 2020 Feb 25. PMID: 32048822.</p></li>
                            <li><p>Tan, S. H., Nguyen, N. T., Chua, Y. C., &amp; Kang, T. G. (2010). Oxygen plasma treatment for reducing hydrophobicity of a sealed polydimethylsiloxane microchannel. <em>Biomicrofluidics, 4</em>(3), 32204. <a href="http://doi.org/10.1063/1.346688">http://doi.org/10.1063/1.346688</a></p></li>
                            <li><p>Thermo Fisher Scientific. (n.d.). <em>Amplex™ red Hydrogen Peroxide/Peroxidase Assay kit</em>. Retrieved July 15, 2021, from <a href="https://www.thermofisher.com/order/catalog/product/A22188#/A22188">https://www.thermofisher.com/order/catalog/product/A22188#/A22188</a>.</p></li>
                            <li><p>Thomsen, M. S., Pölt, P., &amp; Nidetzky, B. (2007). Development of a microfluidic immobilised enzyme reactor. <em>Chem. Commun</em>., <em>24</em>, 2527–2529. <a href="https://doi.org/10.1039/b702115a">https://doi.org/10.1039/b702115a</a></p></li>
                            <li><p>Trantidou, T., Elani, Y., Parsons, E., &amp; Ces, O. Hydrophilic surface modification of PDMS for droplet microfluidics using a simple, quick, and robust method via PVA deposition. <em>Microsyst Nanoengineering; 3</em>, 16091 (2017). <a href="https://doi.org/10.1038/micronano.2016.91">https://doi.org/10.1038/micronano.2016.91</a></p></li>
                            <li><p>Zimmermann, M., Schmid, H., Hunziker, P., Delamarche, E. Capillary pumps for autonomous capillary systems. Lab on a Chip 1, 119-125 (2007). <a href="https://doi.org/10.1039/B609813D">https://doi.org/10.1039/B609813D</a></p></li>
                        </ul>
                        </p>
                    </div>
                </Collapsible>

                <Collapsible open title="Biomarker Detection: Fusion Protein Design">
                    <div className={overallStyles.description}>
                        <p>
                        <ul className={overallStyles.description} style={{marginTop: 0}}>
                            <li><p>Baker, G. B., Bornstein, R. A., Rouget, A. C., Ashton, S. E., van Muyden, J. C., &amp; Coutts, R. T. (1991). Phenylethylaminergic mechanisms in attention-deficit disorder. <em>Biological Psychiatry</em>, <em>29</em>(1), 15-22. doi:0006-3223(91)90207-3 [pii]</p></li>
                            <li><p>Biederman, J. (2005). Attention-deficit/hyperactivity disorder: A selective overview. <em>Biological Psychiatry</em>, <em>57</em>(11), 1215-1220. doi:S0006-3223(04)01100-X [pii]</p></li>
                            <li><p>Faraone, S. V., Bonvicini, C., &amp; Scassellati, C. (2014). Biomarkers in the diagnosis of ADHD--promising directions. <em>Current Psychiatry Reports</em>, <em>16</em>(11), 497-014-0497-1. doi:10.1007/s11920-014-0497-1 [doi]</p></li>
                            <li><p>Feres, M. C., Cintra, F. D., Rizzi, C. F., Mello-Fujita, L., Lino de Souza, A.,A., Tufik, S., &amp; Poyares, D. (2014). Evaluation and validation of a method for determining platelet catecholamine in patients with obstructive sleep apnea and arterial hypertension. <em>PloS One, 9</em>(6), e98407-e98407. doi:10.1371/journal.pone.0098407</p></li>
                            <li><p>Gerra, G., Leonardi, C., Cortese, E., Zaimovic, A., Dell&#39;agnello, G., Manfredini, M., Somaini, L., Petracca, F., Caretti, V., Saracino, M. A., Raggi, M. A, &amp; Donnini, C. (n.d.). Homovanillic acid (HVA) plasma levels inversely correlate with attention deficit-hyperactivity and childhood neglect measures in addicted patients. <em>Journal of Neural Transmission (Vienna, Austria : 1996), 114</em>(12), 1637-1647. doi:10.1007/s00702-007-0793-6 [doi]</p></li>
                            <li><p>Jayamohananan, H., Kumar, M. K. M., &amp; Aneesh, T. P. (2019). 5-HIAA as a potential biological marker for neurological and psychiatric disorders. <em>Advanced Pharmaceutical Bulletin, 9</em>(3), 374-381. doi:10.15171/apb.2019.044 [doi]</p></li>
                            <li><p>Kusaga, A., Yamashita, Y., Koeda, T., Hiratani, M., Kaneko, M., Yamada, S., &amp; Matsuishi, T. (2002). Increased urine phenylethylamine after methylphenidate treatment in children with ADHD. <em>Annals of Neurology, 52</em>(3), 372-374. doi:10.1002/ana.10302 [doi]</p></li>
                            <li><p>Oades, R. D., Sadile, A. G., Sagvolden, T., Viggiano, D., Zuddas, A., Devoto, P., Aase, H., Johansen, E. B., Ruocco, L. A., &amp; Russell, V. A. (2005, February 21). The control of responsiveness in ADHD by catecholamines: Evidence for dopaminergic, noradrenergic and interactive roles. <em>Developmental Science, 8</em>(2), 122-131. doi:DESC399 [pii]</p></li>
                            <li><p>Pliszka, S. R., McCracken, J. T., &amp; Maas, J. W. (1996). Catecholamines in attention-deficit hyperactivity disorder: Current perspectives. <em>Journal of the American Academy of Child and Adolescent Psychiatry</em>. <em>35</em>(3), 264-272. <a href="https://doi.org/10.1097/00004583-199603000-00006">https://doi.org/10.1097/00004583-199603000-00006</a></p></li>
                            <li><p>Scassellati, C., Bonvicini, C., Faraone, S. V., &amp; Gennarelli, M. (2012). Biomarkers and attention-deficit/hyperactivity disorder: A systematic review and meta-analyses. <em>Journal of the American Academy of Child and Adolescent Psychiatry, 51</em>(10), 1003-1019.e20. doi:S0890-8567(12)00605-3 [pii]</p></li>
                            <li><p>Zametkin, A. J., Karoum, F., Rapoport, J. L., Brown, G. L., &amp; Wyatt, R. J. (1984). Phenylethylamine excretion in attention deficit disorder. <em>Journal of the American Academy of Child Psychiatry, 23</em>(3), 310-314. doi:10.1016/s0002-7138(09)60509-4 [doi]</p></li>
                            <li><p>Chushak, Y., &amp; Stone, M. O. (2009). In silico selection of rna aptamers. <em>Nucleic Acids Research, 37</em>(12), 87–95. <a href="https://doi.org/10.1093/nar/gkp408">https://doi.org/10.1093/nar/gkp408</a></p></li>
                            <li><p>Hu, L., Zhang, B.-T., Lu, A., Wang, Y., Yu, Y., &amp; Zhang, G. (2021). Artificial intelligence in Aptamer–Target Binding Prediction. <em>International Journal of Molecular Sciences, 22</em>(7), 3605–3622. <a href="https://doi.org/10.3390/ijms22073605">https://doi.org/10.3390/ijms22073605</a></p></li>
                            <li><p>Ilgu, M., &amp; Nilsen-Hamilton, M. (2016). Aptamers in analytics. <em>The Analyst, 141</em>(5), 1551–1568. <a href="https://doi.org/10.1039/c5an01824b">https://doi.org/10.1039/c5an01824b</a></p></li>
                            <li><p>Keefe, A. D., Pai, S., &amp; Ellington, A. (2010). Aptamers as therapeutics. <em>Nature Reviews Drug Discovery, 9</em>(7), 537–550. <a href="https://doi.org/10.1038/nrd3141">https://doi.org/10.1038/nrd3141</a></p></li>
                            <li><p>Sabri, M. Z., Hamid, A. A., Hitam, S. M., &amp; Rahim, M. Z. (2020). The assessment of three dimensional modelling design for single strand DNA Aptamers for computational Chemistry application. <em>Biophysical Chemistry, 267</em>, 106492–106503. <a href="https://doi.org/10.1016/j.bpc.2020.106492">https://doi.org/10.1016/j.bpc.2020.106492</a></p></li>
                            <li><p>Thiel, K. W., &amp; Giangrande, P. H. (2009). Therapeutic applications of DNA and RNA Aptamers. <em>Oligonucleotides, 19</em>(3), 209–222. <a href="https://doi.org/10.1089/oli.2009.0199">https://doi.org/10.1089/oli.2009.0199</a></p></li>
                            <li><p>Tyagi, S., &amp; Kramer, F. R. (2012). Molecular beacons in diagnostics. <em>F1000 Medicine Reports, 4</em>(10). <a href="https://doi.org/10.3410/m4-10">https://doi.org/10.3410/m4-10</a></p></li>
                            <li><p>Zhuo, Z., Yu, Y., Wang, M., Li, J., Zhang, Z., Liu, J., Wu, X., Lu, A., Zhang, G., &amp; Zhang, B. (2017). Recent advances in Selex technology AND APTAMER applications in biomedicine. <em>International Journal of Molecular Sciences, 18</em>(10), 2142–2160. <a href="https://doi.org/10.3390/ijms18102142">https://doi.org/10.3390/ijms18102142</a></p></li>
                            <li><p>Boulton A. A., &amp; Milward, L. (1971, January 14). Separation, detection and quantitative analysis of urinary β-phenylethylamine. <em>Journal of Chromatography A</em>, <em>57</em>, 287-296. <a href="https://doi.org/10.1016/0021-9673(71)80042-0">https://doi.org/10.1016/0021-9673(71)80042-0</a></p></li>
                            <li><p>Urbina, J., Patil, A., Fujishima, K., Paulino-Lima, I., Saltikov, C., &amp; Rothschild, L. (2019, November 11). A new approach to biomining: Bioengineering surfaces for metal recovery from aqueous solutions. <em>Scientific Reports</em>, <em>9</em>, 16422. <a href="https://doi.org/10.1038/s41598-019-52778-2">https://doi.org/10.1038/s41598-019-52778-2</a></p></li>
                            <li><p>Irsfeld, M., Spadafore, M., Pruess, B. M. (2014, January 28). β-phenylethylamine, a small molecule with a large impact. <em>Webmedcentral, 4</em>(9), 4409. <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904499/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3904499/</a></p></li>
                            <li><p>Sabelli, H., Fink, P., Fawcett, J., Tom, C. (1996, January 1). Sustained antidepressant effect of PEA replacement. <em>The Journal of Neuropsychiatry and Clinical Neurosciences, 8</em>(2), 168-171. <a href="https://doi.org/10.1176/jnp.8.2.168">https://doi.org/10.1176/jnp.8.2.168</a></p></li>
                            <li><p>Eiden, L. E., &amp; Weihe, E. (2011). VMAT2: A dynamic regulator of brain monoaminergic neuronal function interacting with drugs of abuse. <em>Annals of the New York Academy of Sciences</em>, <em>1216</em>, 86-98. doi:10.1111/j.1749-6632.2010.05906.x</p></li>
                            <li><p>Lindemann, L., Ebeling, M., Kratochwil, N. A., Bunzow, J. R., Grandy, D. K., &amp; Hoener, M. C. (2005). Trace amine-associated receptors form structurally and functionally distinct subfamilies of novel G protein-coupled receptors. <em>Genomics</em>, <em>85</em>(3), 372-385. doi:S0888-7543(04)00316-7 [pii]</p></li>
                            <li><p>Marinko, J. T., Huang, H., Penn, W. D., Capra, J. A., Schlebach, J. P., &amp; Sanders, C. R. (2019). Folding and misfolding of human membrane proteins in health and disease: From single molecules to cellular proteostasis. <em>Chemical Reviews</em>, <em>119</em>(9), 5537-5606. doi:10.1021/acs.chemrev.8b00532</p></li>
                            <li><p>Murakawa, T., Hamaguchi, A., Nakanishi, S., Kataoka, M., Nakai, T., Kawano, Y., . . . Okajima, T. (2015). Probing the catalytic mechanism of copper amine oxidase from arthrobacter globiformis with halide ions. <em>The Journal of Biological Chemistry</em>, <em>290</em>(38), 23094-23109. doi:10.1074/jbc.M115.662726</p></li>
                            <li><p>Urbina, J., Patil, A., Fujishima, K., Paulino-Lima, I., Saltikov, C., &amp; Rothschild, L. (2019, November 11). A new approach to biomining: Bioengineering surfaces for metal recovery from aqueous solutions. Retrieved October 17, 2020, from <a href="https://www.ncbi.nlm.nih.gov/pubmed/31712654">https://www.ncbi.nlm.nih.gov/pubmed/31712654</a></p></li>
                            <li><p>Zeng, J., &amp; Spiro, S. (2013). Finely tuned regulation of the aromatic amine degradation pathway in escherichia coli. <em>Journal of Bacteriology</em>, <em>195</em>(22), 5141-5150. doi:10.1128/JB.00837-13</p></li>
                            <li><p>Zucchi, R., Chiellini, G., Scanlan, T. S., &amp; Grandy, D. K. (2006). Trace amine-associated receptors and their ligands. <em>British Journal of Pharmacology</em>, <em>149</em>(8), 967-978. doi:10.1038/sj.bjp.0706948</p></li>
                            <li><p>Chapman-Smith, A., &amp; Cronan Jr, J. E. (1999). Molecular biology of biotin attachment to proteins. <em>The Journal of Nutrition, 129</em>(2), 477–484. <a href="https://doi.org/10.1093/jn/129.2.477s">https://doi.org/10.1093/jn/129.2.477s</a></p></li>
                            <li><p>Cull, M. G., &amp; Schatz, P. J. (2000). Biotinylation of proteins in vivo and in vitro using Small Peptide tags. <em>Methods in Enzymology</em>, (326), 430–440. <a href="https://doi.org/10.1016/s0076-6879(00)26068-0">https://doi.org/10.1016/s0076-6879(00)26068-0</a></p></li>
                            <li><p>Kay, B. K., Thai, S., &amp; Volgina, V. V. (2009). High-throughput biotinylation of proteins. <em>Methods in Molecular Biology</em>, (498), 185–198. <a href="https://doi.org/10.1007/978-1-59745-196-3_13">https://doi.org/10.1007/978-1-59745-196-3_13</a></p></li>
                            <li><p>Wilchek, M., Bayer, E. A., &amp; Livnah, O. (2006). Essentials of biorecognition: The (strept)avidin–biotin system as a model for protein–protein and protein–ligand interaction. <em>Immunology Letters, 103</em>(1), 27–32. <a href="https://doi.org/10.1016/j.imlet.2005.10.022">https://doi.org/10.1016/j.imlet.2005.10.022</a></p></li>
                            <li><p>Amet, N., Lee, H.-F., &amp; Shen, W.-C. (2008). Insertion of the designed helical linker led to increased expression of tf-based fusion proteins. <em>Pharmaceutical Research, 26</em>(3), 523–528. <a href="https://doi.org/10.1007/s11095-008-9767-0">https://doi.org/10.1007/s11095-008-9767-0</a></p></li>
                            <li><p>Argos, P. (1990). An investigation of oligopeptides linking domains in protein tertiary structures and possible candidates for general gene fusion. <em>Journal of Molecular Biology, 211</em>(4), 943–958. <a href="https://doi.org/10.1016/0022-2836(90)90085-z">https://doi.org/10.1016/0022-2836(90)90085-z</a></p></li>
                            <li><p>Aurora, R., Creamer, T. P., Srinivasan, R., &amp; Rose, G. D. (1997). Local interactions in Protein FOLDING: Lessons from the α-Helix. <em>Journal of Biological Chemistry, 272</em>(3), 1413–1416. <a href="https://doi.org/10.1074/jbc.272.3.1413">https://doi.org/10.1074/jbc.272.3.1413</a></p></li>
                            <li><p>Chen, X., Zaro, J. L., &amp; Shen, W.-C. (2013). Fusion protein linkers: Property, design and functionality. <em>Advanced Drug Delivery Reviews, 65</em>(10), 1357–1369. <a href="https://doi.org/10.1016/j.addr.2012.09.039">https://doi.org/10.1016/j.addr.2012.09.039</a></p></li>
                        </ul>
                        </p>
                    </div>
                </Collapsible>

                <Collapsible open title="Protein Optimization">
                    <div className={overallStyles.description}>
                        <p>
                        <ul className={overallStyles.description} style={{marginTop: 0}}>
                            <li><p>CHARMM force field files. MacKerell Lab. (n.d.). Retrieved October 20, 2021, from <a href="http://mackerell.umaryland.edu/charmm_ff.shtml#gromacs">http://mackerell.umaryland.edu/charmm_ff.shtml#gromacs</a>.</p></li>
                            <li><p>European Molecular Biology Laboratory (EMBL-EBI). (2021). PEA : Summary. PDBeChem: Ligand Dictionary (PDB Ligand Chemistry - chemical component dictionary). Retrieved October 13, 2021, from <a href="https://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/complete/PEA">https://www.ebi.ac.uk/pdbe-srv/pdbechem/chemicalCompound/complete/PEA</a>.</p></li>
                            <li><p>Fields G. B. (2002). Introduction to peptide synthesis. Current protocols in protein science, Chapter 18, Unit–18.1. <a href="https://doi.org/10.1002/0471140864.ps1801s26">https://doi.org/10.1002/0471140864.ps1801s26</a> </p></li>
                            <li><p>Foulquier, E. (2020, January 20). Amino acids. IMGT Education. Retrieved October 8, 2021, from <a href="http://www.imgt.org/IMGTeducation/Aide-memoire/_UK/aminoacids/charge/">http://www.imgt.org/IMGTeducation/Aide-memoire/_UK/aminoacids/charge/</a>.</p></li>
                            <li><p>Gao, M., &amp; Skolnick, J. (2012). The distribution of ligand-binding pockets around protein-protein interfaces suggests a general mechanism for pocket formation. Proceedings of the National Academy of Science of the United States of America , 109(10), 3784–3789. <a href="https://doi.org/10.1073/pnas.1117768109">https://doi.org/10.1073/pnas.1117768109</a></p></li>
                            <li><p>Hardinger, S. A. (2017). Aromatic-aromatic interaction (aromatic stacking, pi stacking): Illustrated Glossary of Organic Chemistry . Retrieved October 8, 2021, from <a href="http://www.chem.ucla.edu/~harding/IGOC/A/aromatic_aromatic_interaction.html">http://www.chem.ucla.edu/~harding/IGOC/A/aromatic_aromatic_interaction.html</a>.</p></li>
                            <li><p>Hardinger, S. A. (2017). Steric hindrance: Illustrated Glossary of Organic Chemistry . Retrieved October 8, 2021, from <a href="http://www.chem.ucla.edu/~harding/IGOC/S/steric_hindrance.html">http://www.chem.ucla.edu/~harding/IGOC/S/steric_hindrance.html</a></p></li>
                            <li><p>Huang, PS., Boyken, S. &amp; Baker, D. The coming of age of de novo protein design. Nature 537, 320–327 (2016). <a href="https://doi.org/10.1038/nature19946">https://doi.org/10.1038/nature19946</a></p></li>
                            <li><p>Kaufmann, K. W., Lemmon, G. H., DeLuca, S. L., Sheehan, J. H., &amp; Meiler, J. (2010). Practically Useful: What the Rosetta Protein Modeling Suite Can Do for You. Biochemistry (Easton), 49(14), 2987–2998. <a href="https://doi.org/10.1021/bi902153g">https://doi.org/10.1021/bi902153g</a> </p></li>
                            <li><p>Leiros, H.-K. S., Brandsdal, B. O., Andersen, O. A., Os, V., I Leiros, Helland, R., Otlewski, J., Willassen, N. P., &amp; Smalas, A. O. (2003). Trypsin specificity as elucidated by LIE calculations, X-ray structures and association constant measurements. Doi: 10.2210/pdb1UTM/pdb</p></li>
                            <li><p>Lemkul, J. (2018). Gromacs tutorial. MD Tutorials, GROMACS Tutorial - Protein-Ligand Complex. Retrieved October 20, 2021, from <a href="http://www.mdtutorials.com/gmx/complex/index.html">http://www.mdtutorials.com/gmx/complex/index.html</a>. </p></li>
                            <li><p>Merck KGaA. (2021). Amino acids reference chart. MilliporeSigma. Retrieved October 8, 2021, from <a href="https://www.sigmaaldrich.com/CA/en/technical-documents/technical-article/protein-biology/protein-structural-analysis/amino-acid-reference-chart">https://www.sigmaaldrich.com/CA/en/technical-documents/technical-article/protein-biology/protein-structural-analysis/amino-acid-reference-chart</a>.</p></li>
                            <li><p>National Center for Biotechnology Information (2021). PubChem Compound Summary for CID 6287, Valine. Retrieved October 8, 2021 from <a href="https://pubchem.ncbi.nlm.nih.gov/compound/Valine">https://pubchem.ncbi.nlm.nih.gov/compound/Valine</a></p></li>
                            <li><p>National Center for Biotechnology Information (2021). PubChem Compound Summary for CID 750, Glycine. Retrieved October 8, 2021 from <a href="https://pubchem.ncbi.nlm.nih.gov/compound/Glycine">https://pubchem.ncbi.nlm.nih.gov/compound/Glycine</a></p></li>
                            <li><p>Packer, M., Liu, D. Methods for the directed evolution of proteins. Nat Rev Genet 16, 379–394 (2015). <a href="https://doi.org/10.1038/nrg3927">https://doi.org/10.1038/nrg3927</a> </p></li><li><p>Rohl CA, Strauss CE., Misura KM., Baker D. Protein Structure Prediction Using Rosetta. Methods in Enzymology. 2004;383:66-93. doi:10.1016/S0076-6879(04)83004-0 </p></li>
                            <li><p>Rosetta Commons. (n.d.) Point mutant (&quot;pmut&quot;) scan application, pmut_scan_parallel. Retreived from <a href="https://www.rosettacommons.org/docs/latest/application_documentation/design/">https://www.rosettacommons.org/docs/latest/application_documentation/design/</a></p></li>
                            <li><p>Rosetta Commons. (n.d.). Energy Terms in Rosetta. Retrieved from <a href="https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/score-types">https://www.rosettacommons.org/docs/latest/rosetta_basics/scoring/score-types</a> </p></li>
                            <li><p>Zhang CT, Chou KC. Monte Carlo simulation studies on the prediction of protein folding types from amino acid composition. Biophys J. 1992;63(6):1523-1529.</p></li>
                        </ul>
                        </p>
                    </div>
                </Collapsible>

                <Collapsible open title="Gene Marker Detection: Design of mRNA-Binding CRISPR-Cas System">
                    <div className={overallStyles.description}>
                        <p>
                        <ul className={overallStyles.description} style={{marginTop: 0}}>
                            <li><p>Aoki, M. N., de Oliveira Coelho, B., Góes, L. G. B., Minoprio, P., Durigon, E. L., Morello, L. G., Marchini, F. K., Riediger, I. N., do Carmo Debur, M., Nakaya, H. I., &amp; Blanes, L. (2021, April 27). Colorimetric RT-LAMP SARS-CoV-2 diagnostic sensitivity relies on color interpretation and viral load. <em>Scientific Reports, 11</em>(1), 9026. doi:10.1038/s41598-021-88506-y</p></li>
                            <li><p>Becknell, B., Ching, C., &amp; Spencer, J. D. (2019). The responses of the ribonuclease A superfamily to urinary tract infection. <em>Frontiers in Immunology, 10</em>, 2786-2786. doi:10.3389/fimmu.2019.02786</p></li>
                            <li><p>Bradley, M. S., Boudreau, M.-H., Grenier, C., Huang, Z., Murphy, S. K., &amp; Siddiqui, N. Y. (2019). Urine RNA processing in a clinical setting: Comparison of 3 protocols. <em>Female Pelvic Medicine &amp; Reconstructive Surgery</em>. Retrieved October 3, 2021, from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5975100/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5975100/</a>.</p></li>
                            <li><p>Eun, H. (1996). 3 - nucleases. In <em>H. Eun (Ed.)</em>, Enzymology primer for recombinant DNA technology (pp. 145-232). San Diego: Academic Press. doi:<a href="https://doi.org/10.1016/B978-012243740-3/50006-5">https://doi.org/10.1016/B978-012243740-3/50006-5</a>. Retrieved from <a href="https://www.sciencedirect.com/science/article/pii/B9780122437403500065">https://www.sciencedirect.com/science/article/pii/B9780122437403500065</a></p></li>
                            <li><p>Gotte, G., &amp; Menegazzi, M. (2019). Biological activities of secretory RNases: Focus on their oligomerization to design antitumor drugs. <em>Frontiers in Immunology, 10</em>, 2626-2626. doi:10.3389/fimmu.2019.02626</p></li>
                            <li><p>Lalli, M. A., Langmade, J. S., Chen, X., Fronick, C. C., Sawyer, C. S., Burcea, L. C., Wilkinson, M. N., Fulton, R. S., Heinz, M., Buchser, W. J., Head, R. D., Mitra, R. D., &amp; Milbrandt, J. (2020, October 24). Rapid and extraction-free detection of SARS-CoV-2 from saliva by colorimetric reverse-transcription loop-mediated isothermal amplification. <em>Clinical Chemistry, 67</em>(2), 415-424. doi:10.1093/clinchem/hvaa267
                            Seo, J.-W., Moon, H., Kim, S.-Y., Moon, J.-Y., Jeong, K. H., Lee, Y.-H., Kim, Y.-G., Lee, T.-W., Ihm, C.-G., Kim, C.-D., Chung, B. H., Kim, Y. H., &amp; Lee, S. H. (2017, June 27). Both absolute and relative quantification of urinary mrna are useful for non-invasive diagnosis of acute kidney allograft rejection. <em>PloS one</em>. <em>12</em>(6): e0180045. Retrieved October 3, 2021, from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5487057/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5487057/</a>.</p></li>
                            <li><p>Roman, T., Schmitz, M., Polanczyk, G. V., Eizirik, M., Rohde, L. A., &amp; Hutz, M. H. (2002, January 9). Further evidence for the association between attention‐deficit/hyperactivity disorder and the dopamine‐β‐hydroxylase gene. <em>American Journal of Medical Genetics, 114</em>(2), 154-158. <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajmg.10194">https://onlinelibrary.wiley.com/doi/10.1002/ajmg.10194</a>.</p></li>
                            <li><p>Rommelse, N. N., Altink, M. E., Arias-Vásquez, A., Buschgens, C. J. ., Fliers, E., Faraone, S. V., Buitelaar, J. K., Sergeant, J. A., Oosterlaan, J., &amp; Franke, B. (2008). Differential association between MAOA, ADHD and neuropsychological functioning in boys and girls. <em>American Journal of Medical Genetics. Part B, Neuropsychiatric Genetics, 147B</em>(8), 1524–1530. <a href="https://doi.org/10.1002/ajmg.b.30845">https://doi.org/10.1002/ajmg.b.30845</a>.</p></li>
                            <li><p>Adams, J., Crosbie, J., Wigg, K., Ickowicz, A., Pathare, T., Roberts, W., Malone, M., Schachar, R., Tannock, R., Kennedy, J. L., &amp; Barr, C. L. (2004). Glutamate receptor, ionotropic, N-methyl D-aspartate 2A (GRIN2A) gene as a positional candidate for attention deficit/hyperactivity disorder in the 16p13 region. <em>Molecular Psychiatry, 9</em>, 494–499. <a href="https://www.nature.com/articles/4001455.pdf?origin=ppub](https://www.nature.com/articles/4001455.pdf?origin=ppub">https://www.nature.com/articles/4001455.pdf?origin=ppub](https://www.nature.com/articles/4001455.pdf?origin=ppub</a></p></li>
                            <li><p>Tovo-Rodrigues, L., Rohde, L. A., Menezes, A. M. B., Polanczyk, G. V., Kieling, C., Genro, J. P., Anselmi, L., &amp; Hutz, M. H. (2013). DRD4 rare variants in attention-deficit/hyperactivity disorder (ADHD): Further evidence from a birth cohort study. <em>PLoS ONE, 8</em>(12): e85164. <a href="https://doi.org/10.1371/journal.pone.0085164">https://doi.org/10.1371/journal.pone.0085164</a></p></li>
                            <li><p>Ickowicz, A., Feng, Y., Wigg, K., Quist, J., Pathare, T., Roberts, W., Malone, M., Schachar, R., Tannock, R., Kennedy, J. L., &amp; Barr, C. L. (2006, September 6). The serotonin receptor HTR1B: Gene polymorphisms in attention deficit hyperactivity disorder. <em>American Journal of Medical Genetics Part B, 144B</em>, 121–125. <a href="https://onlinelibrary.wiley.com/doi/10.1002/ajmg.b.30398">https://onlinelibrary.wiley.com/doi/10.1002/ajmg.b.30398</a>.</p></li>
                            <li><p>Lee, J., Laurin, N., Crosbie, J., Ickowicz, A., Pathare, T., Malone, M., Kennedy, J. L., Tannock, R., Schachar, R., &amp; Barr, C. L. (2007, March 19). Association study of the nicotinic acetylcholine receptor α4 subunit gene, CHRNA4, in attention‐deficit hyperactivity disorder. <em>Genes Brain and Behaviour, 7</em>(1), 53-60. <a href="https://onlinelibrary.wiley.com/doi/10.1111/j.1601-183X.2007.00325.x">https://onlinelibrary.wiley.com/doi/10.1111/j.1601-183X.2007.00325.x</a>. </p></li>
                            <li><p>Faraone, S. V., &amp; Mick, E. (2010, February 11). Molecular genetics of attention deficit hyperactivity disorder. <em>Psychiatric Clinics of North America, 33</em>(1), 159-180. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0193953X09001063?via%3Dihub">https://www.sciencedirect.com/science/article/abs/pii/S0193953X09001063?via%3Dihub</a>.</p></li>
                            <li><p>Koonin, E. V., &amp; Makarova, K. S. (2019). Origins and evolution of CRISPR-Cas systems. <em>Philosophical Transactions of the Royal Society B: Biological Sciences, 374</em>(1772). <a href="https://royalsocietypublishing.org/doi/10.1098/rstb.2018.0087">https://royalsocietypublishing.org/doi/10.1098/rstb.2018.0087</a></p></li>
                            <li><p>Fozouni, P., Son, S., Derby, M. D. de L., Knott, G. J., Gray, C. N., D’Ambrosio, M. V., Zhao, C., Switz, N. A., Kumar, G. R., Stephens, S. I., Boehm, D., Tsou, C.-L., Shu, J., Bhuiya, A., Armstrong, M., Harris, A. R., Chen, P.-Y., Osterloh, J. M., &amp; Ott, M. (2020, December 4). Amplification-free detection of SARS-COV-2 with CRISPR-CAS13a and mobile phone microscopy. <em>Cell. 184</em>(2), 323-333.e9. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0092867420316238">https://www.sciencedirect.com/science/article/abs/pii/S0092867420316238</a>.</p></li>
                            <li><p>Barr, C. L., Feng, Y., Wigg, K., Bloom, S., Roberts, W., Malone, M., Schachar, R., Tannock, R., &amp; Kennedy, J. L. (2000, June 30). Identification of DNA variants in the snap-25 gene and linkage study of these polymorphisms and attention-deficit hyperactivity disorder. <em>Molecular Psychiatry, 5</em>, 405–409. <a href="https://www.nature.com/articles/4000733">https://www.nature.com/articles/4000733</a>.</p></li>
                            <li><p>Faraone, S. V., Bonvicini, C., &amp; Scassellati, C. (2014). Biomarkers in the diagnosis of ADHD--promising directions. <em>Current Psychiatry Reports. 16</em>(11):497. doi: 10.1007/s11920-014-0497-1. PMID: 25298126.
                            Shinoda, H., Taguchi, Y., Nakagawa, R., Makino, A., Okazaki, S., Nakano, M., Muramoto, Y., Takahashi, C., Takahashi, I., Ando, J., Noda, T., Nureki, O., Nishimasu, H., &amp; Watanabe, R. (2021, April 19). Amplification-free RNA detection with CRISPR–Cas13. <em>Communications Biology, 4</em>, 476. doi: <a href="https://doi.org/10.1038/s42003-021-02001-8">https://doi.org/10.1038/s42003-021-02001-8</a></p></li>
                            <li><p>Zhang, L., Chang, S., Li, Z., Zhang, K., Du, Y., Ott, J., &amp; Wang, J. (2012). ADHDgene: a genetic database for attention deficit hyperactivity disorder. <em>Nucleic acids research, 40</em>(Database issue), D1003–D1009. <a href="https://doi.org/10.1093/nar/gkr992">https://doi.org/10.1093/nar/gkr992</a></p></li>
                            <li><p>Shekim, W. O., Javaid, J., Davis, J. M., &amp; Bylund, D. B. (1983). Urinary MHPG and HVA excretion in boys with attention deficit disorder and hyperactivity treated with d-amphetamine. <em>Biological Psychiatry. 18</em>(6):707-14. PMID: 6871304.</p></li>
                            <li><p>Baker, G. B., Bornstein, R. A., Douglass, A. B., Van Muyden, J. C., Ashton, S., &amp; Bazylewich, T. L. (1993). Urinary excretion of MHPG and normetanephrine in attention deficit hyperactivity disorder. <em>Molecular and Chemical Neuropathology, 18</em>(1-2):173-8. doi: 10.1007/BF03160031. PMID: 8466590.</p></li>
                            <li><p>RCSB PDB. (n.d.). <em>1UTM</em>. Retrieved October 17, 2021, from <a href="https://www.rcsb.org/structure/1UTM">https://www.rcsb.org/structure/1UTM</a></p></li>
                            <li><p>DeMonte, D., Drake, E. J., Lim, K. H., Gulick, A. M., &amp; Park, S. (2013, June 17). Structure‐based engineering of streptavidin monomer with a reduced biotin dissociation rate. <em>Proteins, 81</em>(9), 1621-33. <a href="https://onlinelibrary.wiley.com/doi/10.1002/prot.24320">https://onlinelibrary.wiley.com/doi/10.1002/prot.24320</a>.</p></li>
                            <li><p>Roxe, D. M. (1990). Urinalysis. In <em>Clinical methods: The history, physical, and Laboratory Examinations</em>. essay, Butterworths. Retrieved October 3, 2021, from <a href="https://www.ncbi.nlm.nih.gov/books/NBK302/">https://www.ncbi.nlm.nih.gov/books/NBK302/</a>.</p></li>
                            <li><p>Abudayyeh, O., &amp; Gootenberg, J. (n.d.) Zhang Lab: Tips and Tricks for Cas13. Retrieved October 20, 2021, from <a href="https://zlab.bio/cas13">https://zlab.bio/cas13</a></p></li>
                        </ul>
                        </p>
                    </div>
                </Collapsible>

                <Collapsible open title="Optical Detector and Application Development">
                    <div className={overallStyles.description}>
                        <p>
                        <ul className={overallStyles.description} style={{marginTop: 0}}>
                            <li><p>Advanced Photonix. (n.d.). PDB-C156 [PDF file]. <a href="https://datasheet.octopart.com/PDB-C156-Advanced-Photonix-datasheet-140068.pdf">https://datasheet.octopart.com/PDB-C156-Advanced-Photonix-datasheet-140068.pdf</a></p></li>
                            <li><p>Arduino. (n.d.). analogRead(). <a href="https://www.arduino.cc/reference/en/language/functions/analog-io/analogread/">https://www.arduino.cc/reference/en/language/functions/analog-io/analogread/</a></p></li>
                            <li><p>Bhattacharyya, A., &amp; Klapperich, C. M. (2007). Design and testing of a disposable microfluidic chemiluminescent immunoassay for disease biomarkers in human serum samples. Biomedical Microdevices, 9, 245. <a href="https://doi.org/10.1007/s10544-006-9026-2">https://doi.org/10.1007/s10544-006-9026-2</a></p></li>
                            <li><p>Digi-Key Electronics. (n.d.). SFH 2430-Z ​​OSRAM Opto Semiconductors Inc. <a href="https://www.digikey.ca/en/products/detail/osram-opto-semiconductors-inc/SFH-2430-Z/1668853">https://www.digikey.ca/en/products/detail/osram-opto-semiconductors-inc/SFH-2430-Z/1668853</a></p></li>
                            <li><p>Foysal, K. H., Seo, S. E., Kim, M. J., Kwon, O. S., &amp; Chong, J. W. (2019). Analyte Quantity Detection from Lateral Flow Assay Using a Smartphone. Sensors, 19(21), 4812. <a href="https://doi.org/10.3390/s19214812">https://doi.org/10.3390/s19214812</a></p></li>
                            <li><p>Ghosh, S., Aggarwal, K. Upaassana, V. T., Nguyen, T., Han, J., &amp; Ahn, C. H. (2020). A new microchannel capillary flow assay (MCFA) platform with lyophilized chemiluminescence reagents for a smartphone-based POCT detecting malaria. Microsystems &amp; Nanoengineering, 6, 5. <a href="https://doi.org/10.1038/s41378-019-0108-8">https://doi.org/10.1038/s41378-019-0108-8</a></p></li>
                            <li><p>Ghosh, S., &amp; Ahn, C. H. (2019). Lyophilization of Chemiluminescent Substrate Reagents for High Sensitive Microchannel-based Lateral Flow Assay (MLFA) in Point-of-Care (POC) Diagnostic System. The Analyst, 144(6), 2109-2119. <a href="https://doi.org/10.1039/C8AN01899E">https://doi.org/10.1039/C8AN01899E</a></p></li>
                            <li><p>Hamamatsu. (2021, January). R12900U_H15710_TPMH1378E [PDF file]. <a href="https://www.hamamatsu.com/resources/pdf/etd/R12900U_H15710_TPMH1378E.pdf">https://www.hamamatsu.com/resources/pdf/etd/R12900U_H15710_TPMH1378E.pdf</a></p></li>
                            <li><p>Hemmig, E., Temiz, Y., Gökçe, O., Lovchik, R. D., &amp; Delamarche, E. (2020). Transposing Lateral Flow Immunoassays to Capillary-Driven Microfluidics Using Self-Coalescence Modules and Capillary-Assembled Receptor Carriers [Abstract]. Analytical Chemistry, 92(1), 940-946. <a href="https://doi.org/10.1021/acs.analchem.9b03792">https://doi.org/10.1021/acs.analchem.9b03792</a></p></li>
                            <li><p>International Commission on Illumination. (n.d.). Colorimetric Illuminants. <a href="https://cie.co.at/publications/colorimetric-illuminants">https://cie.co.at/publications/colorimetric-illuminants</a></p></li>
                            <li><p>Jacinto, M. J., Trabuco, J. R. C., Vu, B. V., Garvey, G., Khodadadi, M., Azevedo, A. M., Aires-Barros, M. R., Chang, L., Kourentzi, K., Litvinov, D., &amp; Willson, R. C. (2018). Enhancement of lateral flow assay performance by electromagnetic relocation of reporter particles. PLOS ONE, 14(3). <a href="https://doi.org/10.1371/journal.pone.0186782">https://doi.org/10.1371/journal.pone.0186782</a></p></li>
                            <li><p>Koczula, K. M., &amp; Gallotta, A. (2016). Lateral flow assays. Essays in Biochemistry, 60(1), 111-120. <a href="https://doi.org/10.1042/EBC20150012">https://doi.org/10.1042/EBC20150012</a></p></li>
                            <li><p>Kranthi. (2013, May 31). How to Use IR LED and Photodiode with Arduino. Robotix. <a href="http://startrobotics.blogspot.com/2013/05/how-to-use-ir-led-and-photodiode-with-arduino.html">http://startrobotics.blogspot.com/2013/05/how-to-use-ir-led-and-photodiode-with-arduino.html</a>.</p></li>
                            <li><p>Kusaga, A., Yamashita, Y., Koeda, T., Hiratani, M., Kaneko, M., Yamada, S., &amp; Matsuishi, T. (2002). Increased urine phenylethylamine after methylphenidate treatment in children with ADHD. <em>Annals of neurology, 52</em>(3), 372–374. <a href="https://doi.org/10.1002/ana.10302">https://doi.org/10.1002/ana.10302</a> </p></li>
                            <li><p>Li, Zhao., Chen, H., &amp; Wang, P. (2019). Lateral Flow Assay Ruler for Quantitative and Rapid Point-of-care Testing. Analyst, 144(10), 3314-3322. <a href="https://doi.org/10.1039/c9an00374f">https://doi.org/10.1039/c9an00374f</a></p></li>
                            <li><p>Lin, B., Guan, Z., Song, Y., Song, E., Lu, Z., Liu, D., An, Y., Zhu, Z., Zhou, L., &amp; Yang, C. (2018). Lateral flow assay with pressure meter readout for rapid point-of-care detection of disease-associated protein [Abstract]. Lab chip, 18(6), 965-970. <a href="https://doi.org/10.1039/c8lc00010g">https://doi.org/10.1039/c8lc00010g</a></p></li>
                            <li><p>Microchip Technology. (2011). MCP3302/04. <a href="https://ww1.microchip.com/downloads/en/DeviceDoc/21697F.pdf">https://ww1.microchip.com/downloads/en/DeviceDoc/21697F.pdf</a></p></li>
                            <li><p>Moyano, A., Serrano-Pertierra, E., Salvador, M., Martínez-García, J. C., Rivas, M., &amp; Blanco-López, M. C. (2020). Magnetic Lateral Flow Immunoassays. Diagnostics, 10(5), 288. <a href="https://doi.org/10.3390/diagnostics10050288">https://doi.org/10.3390/diagnostics10050288</a></p></li>
                            <li><p>Qu, Z., Wang K., Alfranca, G., de la Fuente, J. M., &amp; Cui, D. (2020). A plasmonic thermal sensing based portable device for lateral flow assay detection and quantification. Nanoscale Research Letters, 15(1), 10. <a href="https://doi.org/10.1186/s11671-019-3240-3">https://doi.org/10.1186/s11671-019-3240-3</a></p></li>
                            <li><p>Ramachandran, S., Fu, E., Lutz, B., &amp; Yager, P. (2014). Long-term dry storage of an enzyme-based reagent system for ELISA in point-of-care devices [Abstract]. Analyst, 139, 1456-1462. <a href="https://doi.org/10.1039/C3AN02296J">https://doi.org/10.1039/C3AN02296J</a></p></li>
                            <li><p>Strimbu, K., &amp; Tavel, J. A. (2010). What are biomarkers? Current Opinion in HIV and AIDS, 5(6), 463-466. <a href="https://doi.org/10.1097/COH.0b013e32833ed177">https://doi.org/10.1097/COH.0b013e32833ed177</a></p></li>
                            <li><p>Schwartz, E. (2018, April 30). How to Calculate Light Intensity. Sciencing. <a href="https://sciencing.com/calculate-light-intensity-7240676.html">https://sciencing.com/calculate-light-intensity-7240676.html</a></p></li>
                            <li><p>Vishay. (2011, August 23). TEMD5080X01 [PDF file]. <a href="https://www.vishay.com/docs/81643/temd5080x01.pdf">https://www.vishay.com/docs/81643/temd5080x01.pdf</a></p></li>
                            <li><p>Vishay. (2011, November 23). BPW21R [PDF file]. <a href="https://www.vishay.com/docs/81519/bpw21r.pdf">https://www.vishay.com/docs/81519/bpw21r.pdf</a></p></li>
                            <li><p>Wang, W. (n.d.). Optical Detectors [PowerPoint slides]. University of Washington. <a href="https://depts.washington.edu/mictech/optics/me557/detector.pdf">https://depts.washington.edu/mictech/optics/me557/detector.pdf</a></p></li>
                            <li><p>Weatherspoon, D. (2017). <em>Urine 24-Hour Volume Test</em>. Healthline.</p></li>
                            <li><p>Wu, Z., Fu, Q., Yu, S., Sheng, L., Xu, M., Yao, C., Xiao, W., Li, X., &amp; Tang, Y. (2016). Pt@AuNPs integrated quantitative capillary-based biosensors for point-of-care testing application [Abstract]. Biosensors and Bioelectronics, 85, 657-663. <a href="https://doi.org/10.1016/j.bios.2016.05.074">https://doi.org/10.1016/j.bios.2016.05.074</a></p></li>
                            <li><p>Yan W., Wang, K., Xu H., Huo, X., Jin, Q., &amp; Cui D. (2019). Machine Learning Approach to Enhance the Performance of MNP-Labeled Lateral Flow Immunoassay. Nano-Micro Letters, 11(1), 7. ​​<a href="https://doi.org/10.1007/s40820-019-0239-3">https://doi.org/10.1007/s40820-019-0239-3</a></p></li>
                            <li><p>Zhan, L., Granade, T., Liu, Y., Wei, X., Youngpairoj, A., Sullivan, V., Johnson, J., &amp; Bischof, J. (2020). Development and optimization of thermal contrast amplification lateral flow immunoassays for ultrasensitive HIV p24 protein detection. Microsystems &amp; Nanoengineering, 6, 54. <a href="https://doi.org/10.1038/s41378-020-0168-9">https://doi.org/10.1038/s41378-020-0168-9</a></p></li>
                        </ul>
                        </p>
                    </div>
                </Collapsible>
            </div>
        )
    }
}

export default References;